workflow-management/templates/deployment.yaml

```diff
@@ -58,15 +58,15 @@ spec:
             - name: NEXTFLOW_K8S_RUNASUSER
               value: {{ .Values.appConfig.nextflow_k8s_runAsUser | quote }}
             - name: NEXTFLOW_K8S_SERVICEACCOUNT
-              value: {{ .Values.appConfig.nextflow_k8s_serviceAccount | quote }}
+              value: {{ include "workflow-management.serviceAccountName" . }}
             - name: NEXTFLOW_K8S_NAMESPACE
-              value: {{ .Values.appConfig.nextflow_k8s_namespace | quote }}
+              value: {{ .Release.Namespace }}
+            - name: NEXTFLOW_K8S_RUNNAMESPACE
+              value: {{ include "workflow-management.wesNamespace" . }}
```

Comment:
same hardcoded comment here

Reply:
@lepsalex this was purposely done, since chart should control this configuration. Imagine if the user configured `.Values.appConfig.nextflow_k8s_namespace = namespaceA` but installed the chart to namespace `namespaceB`. Not every environment variable needs a 1-to-1 match with appConfig. I dont see a usecase where the NEXTFLOW_K8S_NAMESPACE needs to be different than the namespace its deployed in. This will definitely decrease human error aswell when configuring the values. 

Also, the `wesNamespace` variable will equal the .Release.Namespace if the value `wes.namespace` is not defined